documentation/modules/appendix_crds.adoc

```diff
@@ -187,15 +187,19 @@ It must have the value `oauth` for the type `KafkaListenerAuthenticationOAuth`.
 |boolean
 |checkIssuer                        1.2+<.<a|Enable or disable issuer checking. By default issuer is checked using the value configured by `validIssuerUri`. Default value is `true`.
 |boolean
+|clientAudience                     1.2+<.<a|Audience to use when making a request to authorization server's token endpoint. Used for inter-broker authentication, and for OAuth over PLAIN with `clientId` and `secret`.
+|string
 |clientId                           1.2+<.<a|OAuth Client ID which the Kafka broker can use to authenticate against the authorization server and use the introspect endpoint URI.
 |string
+|clientScope                        1.2+<.<a|Scope to use when making a request to authorization server's token endpoint. Used for inter-broker authentication, and for OAuth over PLAIN with `clientId` and `secret`.
+|string
 |clientSecret                       1.2+<.<a|Link to Kubernetes Secret containing the OAuth client secret which the Kafka broker can use to authenticate against the authorization server and use the introspect endpoint URI.
 |xref:type-GenericSecretSource-{context}[`GenericSecretSource`]
 |customClaimCheck                   1.2+<.<a|JsonPath filter query to be applied to the JWT token or to the response of the introspection endpoint for additional token validation. Not set by default.
 |string
 |disableTlsHostnameVerification     1.2+<.<a|Enable or disable TLS hostname verification. Default value is `false`.
 |boolean
-|enableECDSA                        1.2+<.<a|Enable or disable ECDSA support by installing BouncyCastle crypto provider. Default value is `false`.
+|enableECDSA                        1.2+<.<a|*The `enableECDSA` property has been deprecated.* Enable or disable ECDSA support by installing BouncyCastle crypto provider. Since version 0.24.0 BouncyCastle crypto provider is no longer needed for ECDSA. As a result ECDSA is always enabled and BouncyCastle is not packaged with Strimzi. Value is ignored.
```

Comment:
I would rather we didn't include any version-specific information here.

The bouncy castle libraries are no longer packaged with Strimzi, so I think you can remove the old description. How about something like this:

> *The `enableECDSA` property has been deprecated.* ECDSA support is always enabled. The BouncyCastle libraries are no longer packaged with Strimzi. Value is ignored.

Reply:
Ok, will change it.